validator/src/cli.rs

```diff
@@ -1387,6 +1399,44 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                         .help("Output display mode")
                 )
         )
+        .subcommand(
+            SubCommand::with_name("repair-whitelist")
+                .about("Manage the validator's repair protocol whitelist")
+                .setting(AppSettings::SubcommandRequiredElseHelp)
+                .setting(AppSettings::InferSubcommands)
+                .subcommand(
+                    SubCommand::with_name("get")
+                        .about("Display the validator's repair protocol whitelist")
+                        .arg(
+                            Arg::with_name("output")
+                                .long("output")
+                                .takes_value(true)
+                                .value_name("MODE")
+                                .possible_values(&["json", "json-compact"])
+                                .help("Output display mode")
+                        )
+                )
+                .subcommand(
+                    SubCommand::with_name("set")
+                        .about("Set the validator's repair protocol whitelist")
+                        .setting(AppSettings::ArgRequiredElseHelp)
+                        .arg(
+                            Arg::with_name("clear")
+                            .long("clear")
+                            .takes_value(false)
+                            .help("Clear the validator's repair protocol whitelist")
+                        )
+                        .arg(
+                            Arg::with_name("whitelist")
+                            .long("whitelist")
+                            .validator(is_pubkey)
+                            .value_name("VALIDATOR IDENTITY")
+                            .multiple(true)
+                            .takes_value(true)
+                            .help("Set the validator's repair protocol whitelist")
```

Comment:
Why do we need these two args?
You can just write
```
repair-whitelist set pk1 pk2 ... pkn
```
to set the pubkeys, and if the trailing list is empty then it would just clear the whitelist.

Reply:
Seems like the parsing with arg validation doesn't handle an empty list. I've updated this to follow the same pattern as the `authorized-voter` command.